tasks/main.yml

```diff
@@ -6,12 +6,36 @@
   when: disk_additional_disks
   tags: ['disk', 'pkgs']
 
+- name: "Install additional fs progs"
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items: "{{ additional_fs_utils|default([]) }}"
+  when: additional_fs_utils is defined
+  tags: ['disk', 'pkgs']
+
+- name: "Get disk alignment for disks"
+  shell: |
```

Comment:
@MiLk Yes, it makes sense, but my python skills are too low to develop an ansible module. :-/